SQLiteDemo/app/src/androidTest/java/com/example/android/sqlitedemo/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example.android.sqlitedemo;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumentation test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
```

Comment:
Remove the classes under /test and /androidTests folder. We don't need tests for the sample.